src/component/DefaultTooltipContent.tsx

```diff
@@ -53,7 +53,7 @@ export interface Props<TValue extends ValueType, TName extends NameType> {
   labelFormatter?: (label: any, payload: ReadonlyArray<Payload<TValue, TName>>) => ReactNode;
   label?: any;
   payload?: ReadonlyArray<Payload<TValue, TName>>;
-  itemSorter?: (item: Payload<TValue, TName>) => number | string;
+  itemSorter?: 'dataKey' | 'value' | ((item: Payload<TValue, TName>) => number | string);
```

Comment:
Were these always usable options? 😅

Reply:
Anything in the payload works. I documented these two as they're the only two that are always populated and make sense to sort by.

Reply:
Array of strings works too since we just feed it to lodash.